pkg/common/credentialmanager/credentialmanager.go

```diff
@@ -199,13 +199,13 @@ func parseConfig(data map[string][]byte, config map[string]*Credential) error {
 			if _, ok := config[vcServer]; !ok {
 				config[vcServer] = &Credential{}
 			}
-			config[vcServer].Password = string(credentialValue)
+			config[vcServer].Password = strings.Split(string(credentialValue), "\n")[0]
```

Comment:
Could you add a unit test as well? Thank you

Reply:
done